xds/internal/xdsclient/internal/internal.go

```diff
@@ -20,6 +20,15 @@ package internal
 
 // The following vars can be overridden by tests.
 var (
-	// NewADSStream is a function that returns a new ADS stream.
+	// GRPCNewClient returns a new gRPC Client.
+	GRPCNewClient any // func(string, ...grpc.DialOption) (*grpc.ClientConn, error)
```

Comment:
i see this in LRS stream PR as well. Which one depend on which?

Reply:
I've included the same code in both the PRs because I had to do that to be able to split them into two PRs. So, whichever PR goes in first will contain the changes, and the other one will not.